Remake/Update of all my previous boards.

CarCar101 opened the pull request:
Palm Brinks slightly stutters during balloonport and Colony 4 has a big stuttering during balloonport. The rest of boards have no stuttering on balloonport.
Tested on Emulator.

Sophilautia commented:
Found 1 issue(s):

../_maps/Sandsea/Sandsea.yaml - 1 issue(s):
The webp file Sandsea does not exist

(The file is called Sanse.webp)

Sophilautia commented:
We also recommend updating your boards that use switches to use the "fixed" variants of backgrounds. You can find them on here: https://fortunestreetmodding.github.io/backgrounds

Sophilautia commented:
What's the state of the balloonport tests with this PR?

CarCar101 commented:
> Found 1 issue(s):
> 
> ../_maps/Sandsea/Sandsea.yaml - 1 issue(s): The webp file Sandsea does not exist
> 
> (The file is called Sanse.webp)

I fixed it. How i do edit the pull request to include those changes?

CarCar101 commented:
> We also recommend updating your boards that use switches to use the "fixed" variants of backgrounds. You can find them on here: https://fortunestreetmodding.github.io/backgrounds

I added the fixed backgrounds to the agency as it uses Alltrades Abbey. Palm Brinks has delfino plaza which i could not find a fixed version. The rest of the boards have no switches.

CarCar101 commented:
> What's the state of the balloonport tests with this PR?

Im not sure what "PR" means, sorry.

Sophilautia commented:
PR as in Pull Request. You need to commit these updates to your own fork of the repository and push them.

CarCar101 commented:
> PR as in Pull Request. You need to commit these updates to your own fork of the repository and push them.
I'm not sure if i did it correctly (I'm new at Github) but i updated the new files in my fork, with the Sandsea.webp for Sandsea and bg108fixed for Agency. I am not allowed to delete the old sandse.webp for some reason.

As for the balloonport tests, i did not experiencie any stuttering in any board i updated except Palm Brinks and Colony 4. Palm Brinks has a small stutter (altrough idk if it's less than 1s, probably not) and Colony 4 does have a big stutter on Balloonports. Enne-San told me that if a board has less than 1s stuttering it passes the tests anywys. Colony 4 does not pass the test im sure eitherway.

Sophilautia commented:
Not sure why you wouldn't be allowed to delete it, it's your fork. And you rename it at least?

CarCar101 commented:
> Not sure why you wouldn't be allowed to delete sandse.webp, it's your fork. Can you rename it at least?

I deleted them. Sorry i was looking at the wrong option lol.

Sophilautia commented:
It looks like the changelogs could use some work.

Sophilautia commented:
Pretty sure changelogs are the last thing that needs fixing and we should be good to merge

CarCar101 commented:
> Pretty sure changelogs are the last thing that needs fixing and we should be good to merge

What are the chanelogs? What do i need to fix about them?
